tools/tensorflow_docs/api_generator/generate_lib.py

```diff
@@ -167,7 +167,7 @@ def title(self):
   @property
   def children(self):
     return sorted(
-        self._children, key=lambda x: (x.full_name.upper(), x.full_name))
+        self._children, key=lambda x: (x.full_name.lower(), x.full_name))
```

Comment:
I would just go with `sorted(x.full_name`.

Can you also run this on tensorflow and show me a diff of what the changes are before and after on the _toc.yaml file?

Reply:
Ok. Check https://gist.github.com/bhack/d62c1fd9b5373fd5d2275f34d3ab0ae1